README.md

```diff
@@ -57,9 +57,25 @@ via [npm (node package manager)](https://github.com/npm/npm)
 
     $ npm install sinon-test
 
-or just add it as a `<script src="dist/sinon-test.js"></script>`
-tag to the html where you write your tests. A pre-built browser
-version can be found in the NPM package under `dist/sinon-test.js`.
+or just add `dist/sinon-test.js` (or for ES6 Modules,
+`dist/sinon-test-es.js`) to where you write your tests.
+
+## Browser usage
+
+In place of the `require` statements indicated below, in the
+browser, you should simply reference the global `sinonTest` after
+including a script tag in your HTML:
+
+```html
+<script src="dist/sinon-test.js"></script>
+```
+
+Or if you are in an ES6 Modules environment (modern browsers only), you
+only need to add an import statement:
+
+```js
+import sinonTestFactory from './sinon-test/node_modules/dist/sinon-test-es.js'
```

Comment:
Two things:
1. Error in your import
2. I'd expand the example to include the script tag

Fix:
```html
<script type="module">
import sinon from './node_modules/sinon/pkg/sinon-esm.js';
import sinonTest from './node_modules/sinon-test/dist/sinon-test-es.js';
const test = sinonTest(sinon);

it('should work', test( function() { 
    pass();
}));
</script>
```

Reply:
P.S. Maybe you could update the examples below in the same manner:
- sinonTestFactory => sinonTest
- sinonTest => test

Then at least the code would align with the usage when coding directly in the browser.

Reply:
Sounds good--do you want me to also change the existing `var`s in the examples to `let` and `const`?

Reply:
No, I only think they make sense in an ES2015 setting, as we are otherwise targetting ES5.1.